scripts/run_mypy_checks_test.py

```diff
@@ -68,12 +71,39 @@ def mock_popen_failure(
             run_mypy_checks, 'EXCLUDED_DIRECTORIES',
             ['dir1/', 'dir2/'])
 
-        def mock_install_mypy_prerequisites_success():
+        def mock_install_mypy_prerequisites_success(ci): # pylint: disable=unused-argument
             return 0
         self.swap_install_success = self.swap(
             run_mypy_checks, 'install_mypy_prerequisites',
             mock_install_mypy_prerequisites_success)
 
+        def mock_popen_user_prefix_error_call(
+                unused_cmd_tokens, *args, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Similarly, can't you use `unused_args` and `unused_kwargs`? I think `args` and `kwargs` are just used by convention--I don't think they have any syntactic meaning

Reply:
I have used `unused_args`, `unused_kwargs` now. Thanks!